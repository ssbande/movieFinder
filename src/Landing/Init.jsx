import React from 'react';
import { Image } from 'semantic-ui-react';
import cinemaScreen from '../images/cinema-screen.png';

const InitRender = () => {
  return <Image src={cinemaScreen} size='medium' centered />
}

export default InitRender;