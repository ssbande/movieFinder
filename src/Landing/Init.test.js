import React from 'react';
import { shallow } from 'enzyme';

import Init from './Init';

describe('Init', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Init />);
    expect(wrapper).toMatchSnapshot();
  })
})