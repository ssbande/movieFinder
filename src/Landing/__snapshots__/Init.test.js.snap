// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Init should render correctly 1`] = `ShallowWrapper {}`;
